src/core/blockchain/types/config.ts

```diff
@@ -102,4 +102,5 @@ export interface IBlockchainConfig {
     };
     defaultOrder: number;
     nameServices?: INameService[];
+    amountToKeepInAccount?: BigNumber;
```

Comment:
not optional - put values in every blockchain
We need to avoid as much as possible optional values in config even if it means to return a 0 value